### PR TITLE
[DEV-1286] Update codeowners

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes

Update codeowners

#### Motivation and Context

Change ownership of frontend directories

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,7 +2,12 @@
...
 
 # Frontend codeowners
+apps/nextjs-website/public/icons/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/public/images/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/_contents/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
 apps/nextjs-website/src/components/ @MarcoPonchia @marcobottaro @tommaso1
 apps/nextjs-website/src/editorialComponents/ @MarcoPonchia @marcobottaro @tommaso1
-apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/lib/types/ @MarcoPonchia @marcobottaro @tommaso1
```
Comment (kin0992) [inline CODEOWNERS:11]: Why not the whole `/lib/` folder?

File: CODEOWNERS [hunk 2]
```diff
@@ -2,7 +2,12 @@
 * @datalek @kin0992
 
 # Frontend codeowners
+apps/nextjs-website/public/icons/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/public/images/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/_contents/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
```
Comment (datalek) [inline CODEOWNERS:8]:
```suggestion
apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
apps/nextjs-website/src/helpers/ @MarcoPonchia @marcobottaro @tommaso1
```

**Review by datalek — APPROVED** [review]
LGTM, I would consider simply using `apps/nextjs-website/src @MarcoPonchia @marcobottaro @tommaso1`.

File: CODEOWNERS (hunk 1 above)
Comment (marcobottaro) [reply CODEOWNERS:11]: Because api.ts will deal with the CMS we want you to take part in the process

**Review by kin0992 — APPROVED** [review]

**Review by datalek — APPROVED** [review]
LGTM

**Review by kin0992 — APPROVED** [review]

**Pull request merged**